### PR TITLE
ss-241: Add footer menu on small screen

### PR DESCRIPTION
**Pull request by MstyslavDmytryk**
ezgif-4-bd93688a4d

ezgif-4-ea6022148c

### CODE REVIEW

File: frontend/src/bundles/common/components/tabs/styles.module.scss [hunk 1]
```diff
@@ -36,3 +38,13 @@
     background-color: var(--color-white-100);
     border-radius: var(--b-2);
 }
+
+@media (max-width: $md) {
+    .tabs {
+        z-index: 99;
+        justify-content: space-around;
+        width: 100vw;
```
Comment (Evcat6) [inline frontend/src/bundles/common/components/tabs/styles.module.scss:46]: Can we use there 100%? To make it more responsive

Comment (MstyslavDmytryk) [reply frontend/src/bundles/common/components/tabs/styles.module.scss:50]: menu have to take all width of the screen always, it doesn't do that when I use %

**Comment by nikita-remeslov** [comment]
I don't like the background and text color. Maybe the selected menu should be on a white background?

**Review by anton-otroshchenko — DISMISSED** [review]

File: frontend/src/bundles/common/components/tabs/styles.module.scss (hunk 1 above)
Comment (Evcat6) [reply frontend/src/bundles/common/components/tabs/styles.module.scss:50]: ok)

**Comment by Evcat6** [comment]
Resolve Conflicts

**Review by Evcat6 — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**